modules/apps/foundation/frontend-taglib/frontend-taglib-chart-sample-web/src/main/java/com/liferay/frontend/taglib/chart/sample/web/internal/display/context/ChartSampleDisplayContext.java

```diff
@@ -139,6 +157,37 @@ private void _initGaugeChartConfig() {
 		_gaugeChartConfig.addColumn(new SingleValueColumn("data1", 85.4));
 	}
 
+	private void _initGeomapConfig() {
+		Map<String, String> range = new HashMap<>();
+
+		range.put("max", "#b2150a");
+		range.put("min", "#ee3e32");
```

Comment:
Hey @julien, would it be possible to type these two fields (max and min) instead of storing them into a Map?